### DISCUSSION TITLE
Timestamp

### DISCUSSION THREAD

**Discussion by Yoseph-Arba** [jddingemanse/awtiCodeDev#35] [Q&A]
When I try to add a new column for Timestamp it works but with warning, so is there any one comment on my code?
The date is written as follows:
1/1/2001
1/2/2001
1/3/2001
.....
My code is.....
josData['Timestamp'] = pd.to_datetime(josData.Datetime, dayfirst=True)
josData

**Comment by jddingemanse** [comment]
> it works but with warning

What does the warning say?
The warning might be related to your date organization. You show part of your date format. Is your format in Month/Day/Year or Day/Month/Year? If it is Month/Day/Year, you should use `josData['Timestamp'] = pd.to_datetime(josData.Datetime)` instead of `josData['Timestamp'] = pd.to_datetime(josData.Datetime, dayfirst=True)`. Including `dayfirst=True` is to let Pandas know that your format is Day/Month/Year. Excluding `dayfirst=True` means that Pandas will assume a Month/Day/Year format.
